### DISCUSSION TITLE
Can monocle connect to a github private user that isn't an organization?

### DISCUSSION THREAD

**Discussion by mcouvill** [change-metrics/monocle#1127] [General]
I have the docker container up and running but it isn't accessing any of my repositories. Not getting error messages either so other than not being an org not sure what the issue could be. Yes, did the token and all repositories option so guessing that since I'm not an org it isn't finding them?

**Comment by morucci** [comment]
Hi, yes if the namespace is not a GitHub organization then the crawler config requires that all repositories are listed. The documentation says: "To crawl repositories from a personal GitHub account, you need to set github_organization to the account name and list repositories under the github_repositories key"
